Source/Core/Core/Movie.h

```diff
@@ -101,7 +101,8 @@ struct DTMHeader
 	u8   bongos;
 	bool bSyncGPU;
 	bool bNetPlay;
-	u8   reserved[13];      // Padding for any new config options
+	bool bPAL60;
+	u8   reserved[12];      // Padding for any new config options
```

Comment:
Even so, this has been there before and probably doesn't belong in this PR anyways. Maybe drop an Issue to keep track of this?